easybuild/easyconfigs/e/ELSI-RCI/ELSI-RCI-0.1.0-intel-2021b.eb

```diff
@@ -0,0 +1,40 @@
+easyblock = 'CMakeMake'
+
+name = 'ELSI-RCI'
+version = '0.1.0'
+
+homepage = 'https://wordpress.elsi-interchange.org/'
+description = """ELSI-RCI provides and enhances open-source software packages
+which iteratively solve or circumvent eigenvalue problems in self-consistent
+field calculations based on the Kohn-Sham density-functional theory.
+"""
+
+toolchain = {'name': 'intel', 'version': '2021b'}
+toolchainopts = {'usempi': False, 'pic': True}
```

Comment:
It does not look like ELSI-RCI uses a maths library (BLAS, MKL, etc.) or MPI. This means that we should use the `intel-compilers` toolchain instead, which will result in renaming to `ELSI-RCI-0.1.0-intel-compilers-2021.4.0.eb`

So here:
```
toolchain = {'name': 'intel-compilers', 'version': '2021.4.0'}
toolchainopts = {'pic': True}
```

Reply:
Done.